### DISCUSSION TITLE
Redirect link to the WI from release notes

### DISCUSSION THREAD

**Discussion by Imran109** [rfennell/AzurePipelines#1125] [Q&A] [answered]
Hi all, just want to know if it is possible to get a link to the work item, so that, we can add it as a `href `to the id of the work item which will redirect to the actual work item on the Azure DevOps.

**Below is the sample code snippet we currently use**
```
   {{#forEach queryWorkItems}}
      *  **{{this.id}}** {{lookup this.fields 'System.Title'}}
   {{/forEach}}
```

**Version:** Generate Release Notes (Crossplatform) **3.71.3 (Latest)**

**Comment by rfennell** [comment, marked as answer]
Yes, there is an example in the [wiki](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task), but it what you need is something like

```
  {{#forEach queryWorkItems}}
      - [{{this.id}}]({{replace this.url "_apis/wit/workItems" "_workitems/edit"}}) - {{lookup this.fields 'System.Title'}}
   {{/forEach}}
```

**Comment by FH-Inway** [comment]
What also works is `{{this._links.html.href}}` (assuming `this` is a `workItem`).
